README.md

```diff
@@ -66,7 +66,7 @@ hzcloud starter-cluster create \
   --name=mycluster \
   --region=us-west-2 \
   --total-memory=0.2 \
-  --hazelcast-version=4.0
+  --hazelcast-version=5.0.2
```

Comment:
This should be `5.0.2-1`

Reply:
Fixed. 

Just a note for anybody looking - this doesn't look very user friendly, adding a version that practically doesn't exist publicly. But Huseyin acknowledged that and we'll create a task for improving this on the backend side. I'll add a link to it here once created.